src/rate.rs

```diff
@@ -572,9 +573,17 @@ impl QuantizerParameters {
   ) -> QuantizerParameters {
     let scale = q57(QSCALE + bit_depth as i32 - 8);
     let quantizer = bexp64(log_target_q + scale);
-    let (offset_u, offset_v) = chroma_offset(log_target_q);
-    let quantizer_u = bexp64(log_target_q + offset_u + scale);
-    let quantizer_v = bexp64(log_target_q + offset_v + scale);
+    let (offset_u, offset_v) = chroma_offset(log_target_q, uv_m_q8);
```

Comment:
It looks like `uv_m_q8` slipped in by a rebase error.
```suggestion
    let (offset_u, offset_v) = chroma_offset(log_target_q);
```